diff --git a/node_modules/@panter/vue-i18next/typings/index.d.ts b/node_modules/@panter/vue-i18next/typings/index.d.ts
index a4f85b9..77d7a3c 100644
--- a/node_modules/@panter/vue-i18next/typings/index.d.ts
+++ b/node_modules/@panter/vue-i18next/typings/index.d.ts
@@ -1,10 +1,10 @@
-import i18next from "i18next";
+import * as i18next from "i18next";
 import Vue, { PluginFunction } from "vue";
 
 declare class VueI18Next {
   constructor(i18next: i18next.i18n, options?: VueI18NextOptions);
   i18next: i18next.i18n;
-  t: i18next.TranslationFunction;
+  t: i18next.TFunction;
   resetVm: ({  }: { i18nLoadedAt: Date }) => void;
   i18nLoadedAt: string;
   onI18nChanged: () => void;
@@ -13,7 +13,7 @@ declare class VueI18Next {
   static version: string;
 }
 
-export interface VueI18NextOptions extends i18next.TranslationOptions {
+export interface VueI18NextOptions extends i18next.TOptions {
   bindI18n?: string;
   bindStore?: string;
   loadComponentNamespace?: boolean;
@@ -36,7 +36,7 @@ declare module "vue/types/options" {
 declare module "vue/types/vue" {
   interface Vue {
     readonly $i18n: VueI18Next;
-    $t: i18next.TranslationFunction;
+    $t: i18next.TFunction;
   }
 }
 
